File: tests/lang/typescript/hello.test.ts
// Hello-world fixture for the TypeScript lane (B88).
// Runs on node's built-in test runner with native type-stripping — no jest, no vitest, no deps.
import test from "node:test";
import assert from "node:assert/strict";
import { hello } from "./hello.ts";

test("typescript lane executes and types strip", () => {
  assert.equal(hello(), "hello, weyland");
});

File: tests/lang/go/hello_test.go
package fixture

import "testing"

// TestHello proves the Go lane runs: toolchain, module resolution, race detector, coverage.
func TestHello(t *testing.T) {
	if got := Hello(); got != "hello, weyland" {
		t.Fatalf("Hello() = %q, want %q", got, "hello, weyland")
	}
}
